README.md

```diff
@@ -212,39 +212,42 @@ This will create a "changeset": a `.md` file containing the release information.
 
 ### Unable to commit
 
-If you get a `command not found` error or a message saying you're using the wrong version of Node when commiting using a GUI (VSCode, GitHub desktop etc), add a `~/.huskyrc` file and load your Node version manager there.
+> [!IMPORTANT]
+> The ~/.huskyrc file has moved to ~/.config/husky/init.sh
+
+If you get a `command not found` error or a message saying you're using the wrong version of Node when commiting using a GUI (VSCode, GitHub desktop etc), add a `~/.config/husky/init.sh` file and load your Node version manager there.
```

Comment:
this is really nice. I wonder if it makes sense to put the note about the file having moved after we first mention it?

```suggestion
If you get a `command not found` error or a message saying you're using the wrong version of Node when commiting using a GUI (VSCode, GitHub desktop etc), add a `~/.config/husky/init.sh` file and load your Node version manager there.

> [!NOTE]
> This used be located in `~/.huskyrc`. If you set that up before, you will need to recreate it at `~/.config/husky/init.sh`.
> ```sh
> mkdir -p ~/.config/husky && cp ~/.huskyrc $_/init.sh
> ```
```